dotCMS/src/main/java/com/dotmarketing/cms/factories/PublicCompanyFactory.java

```diff
@@ -47,6 +47,7 @@
  * @version $Revision: 1.1 $
  * 
  */
+@Deprecated
 public class PublicCompanyFactory extends CompanyUtil {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)